blueprints/component-test/index.js

```diff
@@ -22,13 +39,43 @@ module.exports = {
   locals: function(options) {
     var dasherizedModuleName = stringUtil.dasherize(options.entity.name);
     var componentPathName = dasherizedModuleName;
-    if(options.pod && options.path !== 'components' && options.path !== '') {
+    var testImports = EOL + "import hbs from 'htmlbars-inline-precompile';" + EOL;
+    var testOptions = "integration: true";
+    var friendlyTestDescription = testInfo.description(options.entity.name, "Integration", "Component");
+    var testContent = "assert.expect(1);" + EOL + EOL +
+      "  // Set any properties with this.set('myProperty', 'value');" + EOL +
+      "  // Handle any actions with this.on('myAction', function(val) { ... });" + EOL + EOL +
+      "  this.render(hbs`{{" + dasherizedModuleName + "}}`);" + EOL + EOL +
```

Comment:
We could put both plain usage and block usage in, and leave it to the user to delete the part they're not using. I believe having both tests would pass on initial creation.

This is what I currently have:
```
      "  // Set any properties with this.set('myProperty', 'value');" + EOL +
      "  // Handle any actions with this.on('myAction', function(val) { ... });" + EOL + EOL +
      "  this.render(hbs`{{" + dasherizedModuleName + "}}`);" + EOL + EOL +
      "  assert.equal(this.$().text(), '')" + EOL + EOL +
      "  // Template block usage:"
      "  this.render(hbs`" + EOL
      "    {{#" + dasherizedModuleName + "}}" + EOL
      "      template block text" + EOL
      "    {{/" + dasherizedModuleName + "}}" + EOL
      "  `);" + EOL + EOL +
      "  assert.equals(this.$().text().trim(), 'template block text');"
```